docs/deployment/kubernetes/aws.mdx

```diff
@@ -398,6 +384,72 @@ eksctl create iamidentitymapping --cluster my-app --arn arn:aws:iam::40216744126
 
 In the **--arn** parameter you need to specify your user ARN, which you can find in the IAM dashboard.
 
+## Environment variables
```

Comment:
You can use Reusable snippets for not to duplicate info about k8s envs: https://mintlify.com/docs/content/reusable-snippets